decisive.html

```diff
@@ -53,6 +53,10 @@ <h1 class="logo">
 							<li><label><input type="checkbox"> Max out</label><img src="images/delete-icon.png"></li>
 							<li><label><input type="checkbox"> Relax all cool</label><img src="images/delete-icon.png"></li>
 							<li><label><input type="checkbox"> Shoot some b-ball out by the school</label><img src="images/delete-icon.png"></li>
+							<form class="add-new">
```

Comment:
It looks good!

One thing here to note is that the `<form>` element isn't necessary. The main use of `<form>` is to make it easy for users to actually send form submissions to a website. (You specify the URL and HTTP method (`GET` / `POST`) via the `action` and `method` attributes.)

Besides that use, `<form>`s are also used for semantic purposes -- making it clear based on the HTML that everything inside it is part of a logical form.

You could probably make an argument both ways that `<form>` does or doesn't fit here in a "semantic" sense. If we did use it for semantics it would probably make the most sense to do it around the entire grid -- or at least one in each grid that encapsulates all of the items.

Would it work to replace `<form>` with `<li>` like the other list items? You could still give it the `add-new` class, although some adjustments might be necessary.